packages/cli/src/evaluation/test-runner/__tests__/evaluation-metrics.ee.test.ts

```diff
@@ -0,0 +1,71 @@
+import { EvaluationMetrics } from '../evaluation-metrics.ee';
+
+describe('EvaluationMetrics', () => {
+	test('should roll up metrics correctly', () => {
```

Comment:
I still find the term "roll up" confusing in this context. Could we use "aggregate" instead everywhere?